exonum-java-binding/CHANGELOG.md

```diff
@@ -170,8 +170,8 @@ Parent module and BOM module were released as they are required dependencies to
 
 The first release of Exonum Java Binding.
 
-[Unreleased]: https://github.com/exonum/exonum-java-binding/compare/v0.5...HEAD
-[0.4]: https://github.com/exonum/exonum-java-binding/compare/v0.4...v0.5
+[Unreleased]: https://github.com/exonum/exonum-java-binding/compare/ejb/v0.5.0...HEAD
```

Comment:
With no support of file filters from Github (exonum-java-binding directory) these links will include the light client changes too :upside_down_face: